lib/workers/global/config/parse/file.ts

```diff
@@ -81,3 +91,26 @@ export async function getConfig(env: NodeJS.ProcessEnv): Promise<AllConfig> {
   }
   return config;
 }
+
+export async function deleteConfigFile(env: NodeJS.ProcessEnv): Promise<void> {
+  const configFile = env.RENOVATE_CONFIG_FILE;
+
+  if (is.undefined(configFile) || is.emptyStringOrWhitespace(configFile)) {
+    return;
+  }
+
+  if (!(await fs.pathExists(configFile))) {
```

Comment:
i think this check should be below the ENV check, because most users don't set that ENV and we can skip the existing check all together.